README.md

```diff
@@ -1,75 +1,92 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Rails project: Budget App
 
-> Description the project.
+> Budget calculator website, which has categories of budgets in which you can enter transaction costs for each expense in other to get the total budget price in that particular budget category.
 
+## Screenshot
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+## Link to video
 
-## Live Demo (if available)
```

Comment:
- [ ] It would be better if we include the link to our video presentation. The video presentation must include the following:
  - [ ] The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
  - [ ] The video contains a demo of the project, highlighting all the project features.
  - [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
Done and updated to README.md file.